### DISCUSSION TITLE
Double slash while using model.latex()

### DISCUSSION THREAD

**Discussion by Zhonglu** [MilesCranmer/PySR#225] [Q&A] [answered]
Hi,

I am not sure if it is a bug so I put it here:

How can I remove the `\\` in the latex output? I only need one slash `\` for the convenience of equation examination.

model.latex()
`x_{3} \\left(- \\frac{x_{0}}{x_{4}} + x_{3}\\right) \\left(\\frac{x_{2}}{\\left(x_{1} + 0.703\\right)^{3}} + 1.90\\right)`

Best,

Mike

**Comment by MilesCranmer** [comment, marked as answer]
You can do `print(model.latex())` and they will change in the printout

**Reply by Zhonglu** [reply]
> You can do `print(model.latex())` and they will change in the printout 

Many thanks for the prompt reply!
